### DISCUSSION TITLE
Triggers

### DISCUSSION THREAD

**Discussion by Rudis1261** [flyte/mqtt-io#411] [General]
Is there any chance that we could have digital_inputs toggle a digital_output without additional logic?

I would like to be able to have a physical switch for example to toggle my exterior lights. So with home assistant automating the "automatic" enable when it gets dark. This is doable right now, don't get me wrong, but it relies on HA seeing I pressed the button and switching the lights on. 

I thought it may be nice if, when you press the button, it toggles the lights, and sends both states as mqtt events for HA to reflect that.

**Comment by BenjiU** [comment]
Hi @Rudis1261,

this was already discussed some time ago and the conclusion was, that it is better to have the whole logic at one place and not distributed in different apps.

But if you contribute a working proposel, it will of course be integrated!

Greetings 
Benni

**Comment by Rudis1261** [comment]
100%, fair enough 👍
